e2e-tests/tests/features/dbbackup_pg/22-delete-restore.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1
+kind: TestStep
+timeout: 10
+commands:
+  - command: kubectl -n "${NAMESPACE}" delete dbr --all
+  - command: kubectl -n "${NAMESPACE}" delete pg-restore --all
```

Comment:
Should we delete the pg-restore resource ourselves? Shouldn't we check if it gets deleted automatically by deleting the DBR?

Reply:
A good point, thanks, updated